install_config/registry/accessing_registry.adoc

```diff
@@ -149,8 +149,9 @@ using the following command:
 - The user must have the *system:registry* role. To add this role:
```

Comment:
This line needs to be updated too.

Reply:
Also following `admin` and `system:image-builder` roles are not needed.

Reply:
@nekop 
1. For `The user must have the *system:registry* role. To add this role:` this line needs update too? what changes are required? 

2. For `Also following admin and system:image-builder roles are not needed.` Do you mean to remove the following lines?
```
- For writing or pushing images, for example when using the `docker push` command,
the user must have the *system:image-builder* role. To add this role:
+
----
$ oc adm policy add-role-to-user system:image-builder <user_name>
----
```

Reply:
The "system:" prefix roles are meant for system service account. It's not for normal users.

For image pull, we need "registry-viewer". For push "registry-editor". Other roles are not needed.